database/migrations/2017_01_28_222114_remove_viewed_at_from_contacts.php

```diff
@@ -14,9 +14,9 @@ class RemoveViewedAtFromContacts extends Migration
     public function up()
     {
         Schema::table('contacts', function (Blueprint $table) {
-            $table->dropColumn(
-                'viewed_at'
-            );
+            if ($table->hasColumn('viewed_at')) {
+                $table->dropColumn('viewed_at');
+            }
         });
```

Comment:
Actually `Blueprint::hasColumn` does not exist, you have to write it:
```suggestion
        if (Schema::hasColumn('contacts', 'viewed_at')) {
            Schema::table('contacts', function (Blueprint $table) {
                $table->dropColumn('viewed_at');
            });
        }
```